cmd/kubeadm/test/util.go

```diff
@@ -57,7 +57,7 @@ func SetupMasterConfigurationFile(t *testing.T, tmpdir string, cfg *kubeadmapi.M
 		kind: MasterConfiguration
 		certificatesDir: {{.CertificatesDir}}
 		api:
-		   advertiseAddress: {{.API.AdvertiseAddress}}
+		   advertiseAddress: {{.API.AdvertiseDNSAddress}}
```

Comment:
Is this change intentional? `AdvertiseAddress` should not be the same as `AdvertiseDNSAddress`

Reply:
Nope that's a mistake, thanks for catching.